app/services/imo_api_service.rb

```diff
@@ -2,6 +2,7 @@ class ImoApiService
   IMO_USERNAME = ENV["IMO_USERNAME"]
   IMO_PASSWORD = ENV["IMO_PASSWORD"]
   BASE_URL = "https://sgp.imo.im/api/simple/".freeze
+  USER_FACING_URL = "https://www.nhf.org.bd".freeze
```

Comment:
What is this URL for?  

It doesn't resolve with the `www`, though without the `www` it works.  We should probably remove that so its a valid address.

```
[~/src/simpledotorg/simple-server (rename-reporting-matviews)]> nslookup nhf.org.bd
Server:		172.16.0.1
Address:	172.16.0.1#53

Non-authoritative answer:
Name:	nhf.org.bd
Address: 203.191.33.185

[~/src/simpledotorg/simple-server (rename-reporting-matviews)]> nslookup www.nhf.org.bd
Server:		172.16.0.1
Address:	172.16.0.1#53

** server can't find www.nhf.org.bd: REFUSED

[~/src/simpledotorg/simple-server (rename-reporting-matviews)]>
```

Reply:
I meant to rename `USER_FACING_URL` to something like, `REDIRECT_URL`. It's the url that the user is navigated to if they click on a certain part of the Imo invitation card. It's a required field and has to be whitelisted by Imo. The www version resolves for me. 
```
nslookup www.nhf.org.bd
Server:		192.168.0.1
Address:	192.168.0.1#53

Non-authoritative answer:
Name:	www.nhf.org.bd
Address: 203.191.33.185
```
Also works in browser. I wonder if it was just a blip.

Reply:
Yup I bet it was some caching thing, it works for me tnow. 👍